### PR TITLE
Bug fix: reporting invalid count for insufficient columns in the PRIMARY KEY constraint definition in CREATE TABLE

### PR DESCRIPTION
**Pull request by sanyamsinghal**
- also introduced a map/set for storing invalidCount for each object to avoid duplicate counts

TODO: add automation tests

### CODE REVIEW

File: yb-voyager/cmd/analyzeSchema.go [hunk 1]
```diff
@@ -555,6 +556,7 @@ func checkDDL(sqlInfoArr []sqlInfo, fpath string) {
 			}
 			for _, partitionColumn := range partitionColumnsList {
 				if !slices.Contains(primaryKeyColumnsList, partitionColumn) { //partition key not in PK
+					summaryMap["TABLE"].invalidCount[sqlInfo.objName] = true
```
Comment (priyanshi-yb) [inline yb-voyager/cmd/analyzeSchema.go:559]: can you please add this to the above two `reportCase()` occurrences for `TABLE` in this else-if block.

**Review by priyanshi-yb — APPROVED** [review]
LGTM with one comment.

File: yb-voyager/cmd/analyzeSchema.go (hunk 1 above)
Comment (sanyamsinghal) [reply yb-voyager/cmd/analyzeSchema.go:559]: done, thanks

**Comment by sanyamsinghal** [comment]
Jenkins Pipeline Test Results: https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline/1177/

**Review by amit-yb — APPROVED** [review]
LGTM

**Pull request merged**